cmd/crowdsec-cli/simulation.go

```diff
@@ -27,7 +27,9 @@ func removeFromExclusion(name string) error {
 }
 
 func enableGlobalSimulation() error {
-	*csConfig.Crowdsec.SimulationConfig.Simulation = true
+	tmpBool := true
```

Comment:
to avoid this, I'd suggest using the golang `new()` construct :

```golang
csConfig.Crowdsec.SimulationConfig.Simulation = new(bool)
*csConfig.Crowdsec.SimulationConfig.Simulation = true
```